trade.lic

```diff
@@ -0,0 +1,258 @@
+custom_require.call(%w(common common-items common-money common-travel drinfomon equipmanager))
```

Comment:
Do you plan to document this on elanthipedia?

Reply:
I do, Ill add a link to the header